helm/hpcc/values.schema.json

```diff
@@ -1111,6 +1111,16 @@
         "disabled": {
           "type": "boolean",
           "description": "If true, disable OTel based trace/span generation"
+        },
+        "logSpanStart": {
+          "type": "boolean",
```

Comment:
If https://github.com/hpcc-systems/HPCC-Platform/pull/18025 is merged in first, we'll need to add any new config options here: https://github.com/hpcc-systems/HPCC-Platform/blob/ba7c055b69f8c80523d438d989a6f6bb705751f8/helm/examples/tracing/README.md